### DISCUSSION TITLE
linkID 구하는 방법

### DISCUSSION THREAD

**Discussion by samknp** [storycraft/node-kakao#207] [Q&A] [answered]
joinChannel()할때 OpenLinkComponent(linkID)가 있어야 하는데 보통 우리가 사용하는건 linkURL이잖아요.
linkURL을 이용해서 OpenLinkComponent(linkID)를 구하는 방법이 있을까요?

**Comment by samknp** [comment, marked as answer]
getJoinInfo() 라는 훌륭한 함수가 있네요.
자삭하고 싶은데 질문 삭제가 안되네요.

**Reply by dn1t** [reply]
궁금한 사람이 와서 볼수도 있으니 남겨두는게 좋지 않을까요

**Reply by chiiheonn** [reply]
@jongryongjeong 혹시 getjoininfo 함수 사용법이 어떻게 되나요 계속 실패떠서..

**Reply by samknp** [reply]
> @jongryongjeong 혹시 getjoininfo 함수 사용법이 어떻게 되나요 계속 실패떠서..

        const res = await this.CLIENT.channelList.open.getJoinInfo(url);  
입니다.
여기서 url은 스트링 입니다.

**Reply by MoonJR** [reply]
@jongryongjeong 자답의 도움을 받고 갑니다! 가이드용으로 남겨두는것 강추네요 !
